### PR TITLE
feat: Display the activity views count - EXO-74854 - Meeds-io/MIPs#161

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue [hunk 1]
```diff
@@ -122,6 +122,19 @@
           @click="displayFullContent">
           <span class="pl-6">{{ $t('UIActivity.label.seeMore') }}</span>
         </v-btn>
+        <div
+          v-if="activityViews"
+          :title="activityViewsTooltip"
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue:127]: it should be v-tooltip instead IMO as it is done in news detail

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue (hunk 1 above)
Comment (sofyenne) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue:127]: To follow the guidelines, we use the same navigator tooltip as the activity link title and summary, as confirmed by the PO!

**Review by azayati — APPROVED** [review]
LGTM

**Review by azayati — CHANGES_REQUESTED** [review]
The label displayed by the tooltip should be updated same as content detail, we need two labels "view" and "views"

**Comment by sofyenne** [comment]
> The label displayed by the tooltip should be updated same as content detail, we need two labels "view" and "views"

done!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**